linked-lists/c-or-cpp/merge.cpp

```diff
@@ -0,0 +1,97 @@
+#include <iostream>
+
+template <typename T>
+class LinkedList{
+	public:
+		int size = 0;
+		void pushfront(const T& data);
+		T& front();	
+		void popfront();
+		void display();
+		void merge(const LinkedList<T>& list2) ;
+	private:
+		struct node{
+			T data;
+			node* next;
+			node(const T& data): data(data), next(nullptr){}
+		};
+		node* _head = nullptr;
```

Comment:
Avoid starting the name of any identifier with underscore ("_"), as they are reserved for implementation to use. Using it might cause conflict in case your compiler is also using the same name.

Reply:
Okay i replaced it with "head_" for private naming convention